### ISSUE TITLE
install step copies gmock/gtest files

### ISSUE THREAD

**Issue by Anton-V-K** [matheusgomes28/cmake-google-tests#1]
The approach with `FetchContent` works quite good with one minor drawback: during install step the files of google test library are also copied (they normally aren't needed).
How to reproduce:
1. Add `install(TARGETS multiply_test)` to `fetch-content/tests/CMakeLists.txt`
2. Configure and build normally (in `build` subdir).
3. Install: `cmake --install . --prefix ../install`
4. Observe the content of `../install`: you will find `include\gmock\*`, `include\gtest\*`, `lib\gmock.lib` and other files from gmock/gtest there.

**Comment by matheusgomes28** [comment]
Hello! Thanks for reporting the issue, I'll take a look to understand what's going on a bit more in depth. I know having the unnecessary files is a bit of a pain, but `FetchContent` literally just downloads the git repo. I'll see if there's some non-cumbersome way of only pulling the necessary stuff.

**Comment by Anton-V-K** [comment]
Well, the straighforward fix is simple:
```
...
option(BUILD_GMOCK OFF)     # no need to build gmock - it isn't used
option(INSTALL_GTEST OFF)   # no need to install googletest
FetchContent_MakeAvailable(googletest)
```

**Comment by matheusgomes28** [comment]
@Anton-V-K do you want to create a PR for it? Sorry I haven't had the time to look into this, and since you already know the fix, feel free to fork and PR so I can merge it in :)

**Comment by matheusgomes28** [comment]
After a very long time, I remembered to fix this with #12 . Thanks for pointing the issue out @Anton-V-K
